loopy/schedule/__init__.py

```diff
@@ -1046,7 +1046,8 @@ def insn_sort_key(insn_id):
                 new_insn_ids_to_try = None
 
             new_toposorted_insns = sched_state.insns_in_topologically_sorted_order[:]
-            new_toposorted_insns.remove(insn)
+            new_toposorted_insns = \
```

Comment:
Maybe just add a comment linking to the issue #199, so that no one is taken aback by this way of implementing "list.remove".

Reply:
Done in b9cb3a0f1d64a4a9844469c997270c4c73a194f4